features/step_definitions/integration_steps.rb

```diff
@@ -63,7 +63,8 @@
 
 Then(/^the space host api key is stored in a variable$/) do
   api_key = remote_conjur_secret(space_host_api_key_variable_id)
-  expect(api_key).to be_a(String)
+  # ***TEMP*** Force a failure to verify cleanup upon failure
+  expect(api_key).to be_a(Float)
```

Comment:
based on this, I'm guessing this isn't ready for review yet?

Reply:
Yes, I should have marked this is W.I.P., but I thought I could be quick and get a test and modify under the radar. :)